src/components/views/Registration.js

```diff
@@ -57,11 +57,9 @@ const Registration = () => {
       localStorage.removeItem('errorMessage');
       // post the new user to the server
 
-      const requestBody = JSON.stringify({username, name, password, birthday, gender});
+      const requestBody = JSON.stringify({username:username.trim(), name, password, birthday, gender});
```

Comment:
Try registering a user with username: '     ' (empty spaces) and see what happens (it shouldn't, right?)

Reply:
Error 401 when trying, Username is already taken. Will try to fix